lib/tmdb/client.rb

```diff
@@ -92,8 +97,8 @@ def common_actors_between_movies(movie_one_title, movie_two_title)
         if not_found_message.present?
           OpenStruct.new(not_found_message: not_found_message)
         else
-          movie_one = GuaranteedMovie.find_or_initialize_from_api(movie_one_results.movies.first.tmdb_id)
-          movie_two = GuaranteedMovie.find_or_initialize_from_api(movie_two_results.movies.first.tmdb_id)
+          movie_one = get_movie_data(movie_one_results.movies.first.tmdb_id)
+          movie_two = get_movie_data(movie_two_results.movies.first.tmdb_id)
```

Comment:
We can talk about making these api calls more performant. Maybe there are more `append_to_resutls...` things we can add. I don't want the client to know about the database though unless we're talking about some pretty clear caching.